pkg/cmd/testdata/scripts/issue1832.txtar

```diff
@@ -1,9 +1,9 @@
 [windows] skip 'UNIX only'
 
 # test that chezmoi add succeeds when changing the permissions of an intermediate directory
-exec chezmoi add $HOME/.config/fish/config
-chmod 700 $HOME/.config
```

Comment:
...after `chezmoi add`, chezmoi has created its state in `.config/chezmoi/chezmoistate.boltdb`, so this line was adding a chezmoi dotfile to chezmoi. Changing the directory name in the test from `.config` to `.dir` was the easy fix.